### PR TITLE
login redirection to home page and after succesful sign up shop added…

### PR DESCRIPTION
**Pull request by abdullahcalisir12**
… to link

# Issue Being Addressed
No related issue
Put the issue(s) # that your PR is addressing here. e.g. #15, #29

# Type of PR

Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

[x] Bug Fix
[x] Refactor
[ ] New Feature
[ ] Update to Existing Feature
[ ] Other (state below)

# Description

**For Bug Fixes:** What was the root cause? How do your changes fix the bug effectively?
After succesful sign up for tenant creating the url on the success page had missing /shop.
 
**For Refactors:** What are we refactoring and why? How do your changes address this need for refactoring?
After successful login redirection path changed to Home page
**For New Features:** What feature are we adding? Explain your technical approach to doing this.

**For Feature Updates:** What feature are we updating? Explain your technical approach to doing this.

**For Other:** State what 'type' of PR yours is and elaborate on the PR's technical content.

# How to Test/Reproduce

Provide steps on how one can test your changes here. e.g.

1. Follow create tenant process to see success page
2. Logout and try to login to see redirection works properly

# Screenshots/casts

If applicable, include any relevant screenshot or screencasts of your changes.

# Additional Context

Provide any additional notes that you think would help effective review of your code.

### CODE REVIEW

File: src/router/authRoutes.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ export const authRoutes = [
         .dispatch('auth/ping')
         .then(response => {
           if (response.isAuthenticated) {
-            next({ name: 'MenuCategoryList' });
+            next({ name: 'Home' });
```
Comment (atharva3010) [inline src/router/authRoutes.js:27]: What happens in the case when user has a tenant, he'll still just go to the customer view from where there's no place to reach the tenantCMS view

Comment (abdullahcalisir12) [reply src/router/authRoutes.js:27]: we can add a button to nav bottom when user click, it can send user to tenant related home page if user has tenant.

Comment (atharva3010) [reply src/router/authRoutes.js:27]: Hmm, let's see what @paulchrisluke thinks.

Comment (paulchrisluke) [reply src/router/authRoutes.js:27]: @atharva3010 @abdullahcalisir12 can we just give tenants the url and have them save that as a bookmark for now?

**Comment by paulchrisluke** [comment]
After I finish making a tenant I click on adding menu items but do not get redirected to that portion. Is that intended?

File: src/router/authRoutes.js (hunk 1 above)
Comment (abdullahcalisir12) [reply src/router/authRoutes.js:27]: it's possible

**Comment by abdullahcalisir12** [comment]
> After I finish making a tenant I click on adding menu items but do not get redirected to that portion. Is that intended?

It's done now it's redirected to categories page

**Pull request merged**